lib/link.ts

```diff
@@ -1,8 +1,8 @@
 import type * as monaco from 'monaco-types'
 import type * as ls from 'vscode-languageserver-protocol'
 
-import { getMonaco } from './monaco.js'
 import { fromRange, toRange } from './range.js'
+import { URI } from 'vscode-uri'
```

Comment:
ESLint is going to complain about unsorted imports. You can fix all ESLint issues using:

```sh
npx eslint --fix .
```